nrf70_bm_lib/include/nrf70_bm_lib.h

```diff
@@ -249,7 +278,7 @@ typedef void (*nrf70_scan_result_cb_t)(struct nrf70_scan_result *entry);
  * @retval 0 If the operation was successful.
  * @retval -1 If the operation failed.
  */
-int nrf70_bm_init(uint8_t *mac_addr);
+int nrf70_bm_init(uint8_t *mac_addr, struct nrf70_regulatory_info *reg_info);
```

Comment:
This changes the API. We 'd have to update developers (E.g. release notes) on migration to this release..

Reply:
Sure, but we don't have that concept for this repo, I can add one.